microservices/cart/src/queries/repositories.py

```diff
@@ -0,0 +1,170 @@
+"""
+Copyright (C) 2021 Clariteia SL
+This file is part of minos framework.
+Minos framework can not be copied and/or distributed without the express permission of Clariteia SL.
+"""
+from __future__ import (
+    annotations,
+)
+
+from typing import (
+    NoReturn,
+)
+from uuid import (
+    UUID,
+)
+
+from minos.common import (
+    FieldDiff,
+    MinosConfig,
+    MinosSetup,
+)
+from sqlalchemy import (
+    and_,
+    create_engine,
+)
+from sqlalchemy.orm import (
+    sessionmaker,
+)
+
+from .models import (
+    CART_ITEM_TABLE,
+    CART_TABLE,
+    META,
+    CartDTO,
+    CartItemDTO,
+)
+
+
+class CartRepository(MinosSetup):
```

Comment:
I would name this class as `CartQueryRepository` to avoid possible confusions with the Cart's events repository.

Reply:
Done!